src/test/scala-2.11/markets/participants/TestOrderCanceler.scala

```diff
@@ -18,52 +18,34 @@ package markets.participants
 import akka.actor.{Props, ActorRef}
 import akka.agent.Agent
 
-import markets.Cancel
 import markets.orders.Order
+import markets.participants.strategies.{TestOrderCancellationStrategy, OrderCancellationStrategy}
 import markets.tickers.Tick
 import markets.tradables.Tradable
 
 import scala.collection.mutable
-import scala.util.Random
+import scala.concurrent.duration.FiniteDuration
+import scala.concurrent.ExecutionContext.Implicits.global
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Imports should be sorted alphabetically](https://www.codacy.com/app/davidrpugh/markets-sandbox/file/1705050679/issues/source?bid=2921361&fileBranchId=3046267#l28)